src/debugger/Stepper.js

```diff
@@ -23,18 +23,11 @@ export class Stepper {
   }
   _stepStartData: StoppedData;
 
-  isComplete(ast: BabelNode): boolean {
+  isComplete(ast: BabelNode, currentStackSize: number): boolean {
     invariant(false, "Abstract method, please override");
   }
-}
-
-export class StepIntoStepper extends Stepper {
-  constructor(filePath: string, line: number, column: number) {
-    super(filePath, line, column);
-  }
 
-  // Override
-  isComplete(ast: BabelNode): boolean {
+  isValidStopLocation(ast: BabelNode) {
```

Comment:
This method will return true even column changed. Do you plan to support column stepping?